tests/unit/backend/data/test_GroceryService.py

```diff
@@ -1047,6 +1059,11 @@ def test_fetch_grocery_list_diffcal_output(self):
                 .buildList()
             )
             diffCalOutputFilename = self.instance._createDiffcalOutputWorkspaceFilename(groceryList[0])
+            wrapLeftovers = WrapLeftovers()
+            wrapLeftovers.initialize()
+            wrapLeftovers.setPropertyValue("InputWorkspace", self.sampleWS)
+            wrapLeftovers.setPropertyValue("Filename", self.sampleTarWsFilePath)
+            wrapLeftovers.execute()
```

Comment:
Could you call here, and above, using the function syntax?
``` python
WrapLeftovers(
    InputWorkspace=self.sampleWS,
    Filename=self.sampleTarWsFilePAth,
)
```
You will need to change the import above, to
``` python
import snapred.backend.recipe.algorithm
from mantid.simpleapi import (
    ...
   the stuff already there,
   ...
   WrapLeftovers,
   ReheatLeftovers,
)
```